quilt/test/utils.py

```diff
@@ -41,12 +48,17 @@ def setUp(self):
         self.auth_patcher = patch('quilt.tools.command._create_auth', lambda: None)
         self.auth_patcher.start()
 
+        self._store_dir = test_store_dir()
```

Comment:
Need to delete the dir in `tearDown` (but test it to make sure it doesn't do anything stupid like delete your home dir :) )